backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -180,6 +182,7 @@ private CloneImageGroupVolumesStructureCommandParameters buildCloneImageGroupVol
         p.setDestImageGroupId(getImageGroupId());
         p.setEndProcedure(EndProcedure.COMMAND_MANAGED);
         p.setJobWeight(Job.MAX_WEIGHT);
+        p.setLiveMigration(true);
```

Comment:
that's also ok but shouldn't we check the parent command as we do elsewhere to detect this?

Reply:
never mind, it's ok this way